examples/src/java/com/twitter/heron/examples/streamlet/IntegerProcessingTopology.java

```diff
@@ -16,10 +16,10 @@
 
 import java.util.concurrent.ThreadLocalRandom;
 
+import com.twitter.heron.common.basics.ByteAmount;
```

Comment:
Can we pl not do this. I specifically want no dep with anything heron other than api.